### DISCUSSION TITLE
hover whole row, select and click-Event

### DISCUSSION THREAD

**Discussion by MORBH177** [Akascape/CTkTable#16] [General]
Hi Akascape, 

nice Table widget you created here. 

Do you or does someone knew how i can do a hover over a whole row and bind an on-click Event to select an entire row?
Is it possible to bind Actions regardless, where i click in the row, so the on-click Event will trigger and deliveres all the data of the row and the rows index?
It shall work like in TreeView widgets from ttk. 

I hope i explained my issue properly and someone can help me with that.

**Comment by Akascape** [comment]
@MORBH177 
> Do you or does someone knew how i can do a hover over a whole row and bind an on-click Event to select an entire row?

Use `edit_row(row_num, hover=True)` to enable the hover_effect in that row.

>Is it possible to bind Actions regardless, where i click in the row, so the on-click Event will trigger and deliveres all the data of the row and the rows index?

Please refer this example here: https://github.com/Akascape/CTkTable/issues/6#issuecomment-1570224204
